src/services/account/docs/index.js

```diff
@@ -0,0 +1,74 @@
+module.exports = {
+	accountService: {
+		description: "A service for accounts",
+		find: {
+			summary: "find"
+		},
+		create: {
+			summary: "create"
+		},
+		remove: {
+			summary: "remove"
+		},
+		update: {
+			summary: "update"
+		},
+		patch: {
+			summary: "patch"
+		},
+		get: {
+			summary: "get"
+		},
+		CustomJWTService: {
+			create: {
+				parameters: [
+					{
+						description: "Userdata for the User to be created",
```

Comment:
No User will be created ...
The Service gets an userId and will give you a valid JWT in return ...